cmd/ocm/gcp/create-wif-config.go

```diff
@@ -137,6 +147,23 @@ func promptProjectId() error {
 	return nil
 }
 
+func promptVersion() error {
+	const projectIdHelp = "The OCP version to configure the wif-config for."
```

Comment:
```suggestion
	const versionHelp = "The OCP version to configure the wif-config for."
```
I believe this may have originally been a name that was copied from promptProjectId. I suggest renaming it to improve readibility.

Reply:
We should specify in the help message that the user may leave the field empty to use the latest wif version.